packages/netlify-cms-backend-azure/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "netlify-cms-backend-azure",
+  "description": "Azure DevOps backend for Netlify CMS",
+  "version": "1.0.0",
+  "license": "MIT",
+  "repository": "https://github.com/netlify/netlify-cms/tree/master/packages/netlify-cms-backend-azure",
+  "bugs": "https://github.com/netlify/netlify-cms/issues",
+  "module": "dist/esm/index.js",
+  "main": "dist/netlify-cms-backend-azure.js",
+  "keywords": [
+    "netlify",
+    "netlify-cms",
+    "backend",
+    "azure",
+    "devops"
+  ],
+  "sideEffects": false,
+  "scripts": {
+    "develop": "yarn build:esm --watch",
+    "build": "cross-env NODE_ENV=production webpack",
+    "build:esm": "cross-env NODE_ENV=esm babel src --out-dir dist/esm --ignore **/__tests__ --root-mode upward --extensions \".js,.jsx,.ts,.tsx\""
+  },
+  "dependencies": {
+    "common-tags": "^1.8.0",
+    "js-base64": "^2.5.1",
```

Comment:
I think we're using `v3` of `js-base64`

Reply:
Ran yarn upgrade and it's gone. Hope to push again before EOD